content/variables/using-environment-variables.md

```diff
@@ -7,6 +7,24 @@ aliases:
 
 To access a variable, add the `$` symbol in front of its name. For example, access `API_TOKEN` by using `$API_TOKEN`. Note that it is required to use quotation marks with multi-line variables when you are referencing them in custom scripts.
 
+## Accessing self-defined variables across the workflow
```

Comment:
I like GH version more

Reply:
- there is no such thing as "self-defined" variables
- can we change and use **setting**, not **accessing** variables?
- one example to combine settings and using variable in next step
- also like problem statement from GH doc
```
You can make an environment variable available to any subsequent steps in a workflow job by defining or updating the environment variable and writing this to the GITHUB_ENV environment file. 
```

Reply:
I didn't add a specific example, as this doc is for yaml and UI both, thus it seemed better to leave it as simplified as possible.

Reply:
ok, let's use one line examples